app/templates/cookies/cookie_settings.html

```diff
@@ -25,7 +25,11 @@
     "type": "error",
     "classes": "dm-cookie-settings__error",
     "titleText": "There was a problem saving your settings",
-    "text": "Please select 'Yes' or 'No'.",
+    "html": '<ul class="govuk-list govuk-error-summary__list">
+      <li>
+        <a href="#dm-cookie-settings">Please select \'Yes\' or \'No\'.</a>
+      </li>
+    </ul>',
     "attributes": [("id", "dm-cookie-settings-error")]
   }) }}
```

Comment:
We might actually be able to replace this `dmAlert` with the [govukErrorSummary component](https://design-system.service.gov.uk/components/error-summary/): 

```
{% from "components/error-summary/macro.njk" import govukErrorSummary %}

{{ govukErrorSummary({
      "titleText": "There was a problem saving your settings",
      "errorList": [{
        "text": "Please select \'Yes\' or \'No\'.",
        "href": "#dm-cookie-settings"
      }],
      "classes": "dm-cookie-settings__error" 
    }) }}
```